packages/appstore/src/components/account/account.tsx

```diff
@@ -1,54 +1,307 @@
 import React from 'react';
-import { Text } from '@deriv/components';
-import { Localize } from '@deriv/translations';
+import { Text, Button, Icon, Money } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
 import WalletIcon from 'Assets/svgs/wallet';
-import { getCurrencyDisplayCode } from '@deriv/shared';
+import {
+    getCurrencyDisplayCode,
+    isMobile,
+    getCurrencyName,
+    getCFDAccountDisplay,
+    isBot,
+    formatMoney,
+} from '@deriv/shared';
+import classNames from 'classnames';
+import { RouteComponentProps } from 'react-router';
 
-type TOptionsAccountprops = {
-    account_icon: string;
-    account_title?: string;
-    account_number?: string;
-    account_balance?: string;
+type Icountry_standpoint = {
+    is_united_kingdom: boolean;
+    is_isle_of_man: boolean;
+};
+
+type Igeolocation = {
+    region: any;
+    sequence: number;
+};
+type Iserver_info = {
+    geolocation: Igeolocation;
+};
+type Iserver = {
+    server_info: Iserver_info;
+};
+
+type TOptionsAccountprops = RouteComponentProps & {
+    currency_icon: string | undefined;
+    loginid_text: string;
+    balance?: string;
     account_button?: string;
     currency?: string;
+    display_type: string;
+    has_balance?: boolean;
+    has_reset_balance?: boolean;
+    is_disabled?: boolean;
+    is_virtual?: boolean;
+    title?: string;
+    country_standpoint: Icountry_standpoint;
+    is_eu?: string;
+    market_type?: string;
+    server?: Iserver;
+    sub_account_type?: string;
+    has_error?: string;
+    platform?: string;
+    is_dark_mode_on?: string;
+    shortcode?: string;
+    should_show_server_name?: string;
+    onClickResetVirtualBalance: () => void;
+    selected_loginid?: string;
+    redirectAccount?: () => void;
+    activeAccount?: string;
+    onClickDeposit?: () => void;
+    switchAccountModal?: () => void;
+    isModal?: boolean;
```

Comment:
this props should has snake_case format due to the deriv-app naaming conventions
```suggestion
    is_modal?: boolean;
```